include/libtorrent/torrent_handle.hpp

```diff
@@ -1265,6 +1281,24 @@ namespace aux {
 		// client builds.
 		std::shared_ptr<torrent> native_handle() const;
 
+		// Returns true if the torrent is active in its session; returns false if
+		// it has been removed.
+		//
+		// A ``torrent_handle`` remains valid for some time after
+		// ``session::remove_torrent()`` is called. It will become invalid only
+		// after all libtorrent tasks (such as I/O tasks) release their references
+		// to the torrent. Until this happens, ``torrent_handle::is_valid()`` will
+		// return true, and other operations such as ``torrent_handle::status()``
+		// will succeed.
+		//
+		// ``torrent_handle::is_in_session()`` can be used to figure out if
+		// the call to ``session::remove_torrent()`` has been made. It returns true
+		// before ``torrent_removed_alert`` is posted, and false afterward.
+		//
+		// Note that this function posts a message to the libtorrent control thread
+		// and waits for the result, unlike ``is_valid()`` which returns instantly.
```

Comment:
the thread is referred to as the "network thread", but I think there's precedence for documenting these kinds of calls. I think they're just called "blocking"